drivers/spi/spi_nxp_lpspi/spi_nxp_lpspi_dma.c

```diff
@@ -50,6 +51,8 @@ static void spi_mcux_dma_callback(const struct device *dev, void *arg, uint32_t
 	if (channel == dma_data->dma_tx.channel) {
 		/* this part of the transfer ends */
 		dma_data->status_flags |= LPSPI_DMA_TX_DONE_FLAG;
+		/* Disable continuous mode when TX done. */
+		base->TCR &= ~LPSPI_TCR_CONT_MASK;
```

Comment:
I think this is not going to control the chip select correctly, because the way this dma driver currently works is that there will be a callback after every dma chunk, so I think this change will cause the chip select to deassert during the transfer.

Reply:
I did not see CS is deassertted from the signal wave during loopback test. you can try yourself.